controllers/topics.controllors.js

```diff
@@ -0,0 +1,17 @@
+//require model
+const { selectTopicsFromDatabase } = require("../models/topics.model");
+
+exports.getTopics = (req, res) => {
+  selectTopicsFromDatabase().then((topics) => {
+    res.status(200).send(topics);
```

Comment:
Sending back the topics data is great here but it would be nice to give it a descriptive key so when a user accesses the body they see the topics key with all the data. ie {topics: [returned data array here]}